dicodile/tests/test_config.py

```diff
@@ -1,5 +1,10 @@
 from pathlib import Path
-from dicodile.data.home import get_data_home
+
+from dicodile.config import DATA_HOME, get_data_home
+
+
+def test_DATA_HOME():
+    assert DATA_HOME == Path("/tmp/dicodile/dicodile")
```

Comment:
Actually, I forgot that we already tested `get_data_home` which is used internally for `DATA_HOME` so I am not sure it is worth adding this test.
Sorry about the change of mind.

Reply:
Ok, fixed!